gtsfm/two_view_estimator.py

```diff
@@ -65,6 +68,9 @@ class TwoViewEstimationReport:
     U_error_deg: Optional[float] = None
     i2Ri1: Optional[Rot3] = None
     i2Ui1: Optional[Unit3] = None
+    reproj_error_gt_model: Optional[np.ndarray] = None
```

Comment:
we could use some explanation of these metrics here. maybe a one liner for each. is gt_model == gt_mesh?